tests/arvenums.c

```diff
@@ -0,0 +1,511 @@
+#include <arv.h>
```

Comment:
It is a great idea to implement more unit testing. But it should integrate with the unit testing framework. Have a lookt at evaluator.c for example.

Also, I think you went to far in the number of tests you have implemented. I'm not sure testing every defined pixel formats is useful. The tests should only exercize the bit per pixel API.